bims/models/auth.py

```diff
@@ -93,3 +99,45 @@ def do_logout(sender, user, request, **kwargs):
 
 
 user_logged_out.connect(do_logout)
+
+
+@receiver(user_signed_up, dispatch_uid="user_signed_up")
+def user_signed_up_(request, user, **kwargs):
+    username = request.POST.get('username')
+    user_email = request.POST.get('email')
+    try:
+        site = Site.objects.first()
+        bims_site_name = site.name
+    except Site.DoesNotExist:
+        bims_site_name = 'Biodiversity Information System'
+
+    try:
+        site_setting = SiteSetting.objects.first()
```

Comment:
you can get the value easily by doing this : `preferences.SiteSetting.default_team_name`
you only need to import this : `from preferences import preferences`